package-lock.json

```diff
@@ -743,20 +743,28 @@
     },
     "node_modules/@babel/plugin-syntax-bigint": {
       "version": "7.8.3",
-      "resolved": "https://registry.npmjs.org/@babel/plugin-syntax-bigint/-/plugin-syntax-bigint-7.8.3.tgz",
-      "integrity": "sha512-wnTnFlG+YxQm3vDxpGE57Pj0srRU4sHE/mDkt1qv2YJJSeUAec2ma4WLUnUPeKjyrfntVwe/N6dCXpU+zL3Npg==",
+      "resolved": "https://packages.atlassian.com/api/npm/npm-remote/@babel/plugin-syntax-bigint/-/plugin-syntax-bigint-7.8.3.tgz",
```

Comment:
Wonder why all these are now pointing to atlassian instead of npm 🤔

Reply:
It looks like it's using the registry configured from artifactory :( I'll look into creating an npm work profile, so it doesn't change to this.